runtime/autoload/provider/clipboard.vim

```diff
@@ -1,6 +1,12 @@
 " The clipboard provider uses shell commands to communicate with the clipboard.
 " The provider function will only be registered if a supported command is
 " available.
+if exists('s:loaded_provider_clipboard')
+  finish
+endif
+
+let s:loaded_provider_clipboard = 1
```

Comment:
For reason mentioned here: https://github.com/neovim/neovim/pull/4150#discussion-diff-51673063 I think we should not do this. What happens when user sets this _and_ sets clipboard? Too confusing.

We should instead make the message less annoying. I'm working on that.